cmd/resource/status/docs/commands/events.md

```diff
@@ -0,0 +1,22 @@
+## events
+
+[Alpha] Poll the cluster until all provided resources have become Current and list the status change events.
+
+### Synopsis
+
+[Alpha] Poll the cluster for the state of all the provided resources until either they have all become 
+Current or the timeout is reached. The output will be status change events.
+
+The list of resources which should be polled are provided as manifests either on the filesystem or
+on StdIn. 
+
+  DIR:
+    Path to local directory. If not provided, input is expected on StdIn.
+
+### Examples
+
+    # Read resources from the filesystem and wait up to 1 minute for all of them to become Current
+    resource status events my-dir/
```

Comment:
nit: these should probably be `kustomize resource` right?  the `config` cobra commands support replacing these strings with arbitrary values for better documentation in composition.